modules/tinymce/src/plugins/table/test/ts/browser/TwoCellsSelectionTest.ts

```diff
@@ -0,0 +1,113 @@
+import { Mouse, UiFinder } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyDom, TinyHooks, TinySelections } from '@ephox/mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/table/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
+
+const enum Direction {
+  Row,
+  Column
+}
+
+describe('browser.tinymce.plugins.table.TwoCellsSelectionTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'table',
+    indent: false,
+    base_url: '/project/tinymce/js/tinymce',
+  }, [ Plugin, Theme ]);
+
+  const setup = (editor: Editor, colgroup: boolean, direction: Direction) => {
+    editor.setContent(
+      '<table>' +
+      (colgroup ? '<colgroup><col /><col /><col /></colgroup>' : '') +
+      '<tbody>' +
+      '<tr><td>A1</td><td>B1</td><td>C1</td></tr>' +
+      '<tr><td>A2</td><td>B2</td><td>C2</td></tr>' +
+      '</tbody>' +
+      '</table>'
+    );
+    const path = colgroup ? [ 0, 1, 1, 1, 0 ] : [ 0, 0, 1, 1, 0 ];
+    TinySelections.setCursor(editor, path, 1);
+    const startCell = UiFinder.findIn(TinyDom.body(editor), 'td:contains(B2)').getOrDie();
+    const endCellSelector = direction === Direction.Column ? 'td:contains(B1)' : 'td:contains(A2)';
+    const endCell = UiFinder.findIn(TinyDom.body(editor), endCellSelector).getOrDie();
+
+    // Drag over the 2 cells to select them
+    Mouse.mouseDown(startCell);
```

Comment:
Is this the right order? Wouldn't it mouse-over first

Reply:
Have fixed this as discussed.